elastic/beats
# [Heartbeat] Fix testing pipeline for Windows

Fix windows test pipeline by reflecting browser monitor build constraints in the include file



## Proposed commit message



## Checklist




## Disruptive User Impact


N/A


## How to test this PR locally


- Run `mage update` and check `x-pack/heartbeat/include/list.go` is constrained to `linux || darwin || synthetics` build tags.

## Related issues


Closes https://github.com/elastic/beats/issues/40502.
